src/components/10-atoms/toggle-switch/index.scss

```diff
@@ -0,0 +1,57 @@
+$toggle-switch-height: 30px;
+$toggle-switch-animation: 0.4s ease;
+
+:host {
+  display: inline-block;
+  height: $toggle-switch-height;
+}
+
+.a-toggle-switch {
+  display: inline-block;
+  height: $toggle-switch-height;
+
+  &__input {
+    display: none;
+
+    &:checked {
+      & + .a-toggle-switch__slider {
+        background-color: $color-malachite;
+      }
+
+      & + .a-toggle-switch__slider::before {
+        transform: translateX(20px);
+      }
+
+      &:disabled + .a-toggle-switch__slider {
+        background-color: #9FD9B4;
+      }
+    }
+
+    &:disabled + .a-toggle-switch__slider {
+      background-color: #F5F5F5;
+    }
+  } 
+
+  &__slider {
+    display: inline-block;
+    position: relative;
+    height: $toggle-switch-height;
+    width: 50px;
+    border-radius: 18px;
+    background-color: $color-light-grey;
+    cursor: pointer;
+    transition: $toggle-switch-animation;
+
+    &::before {
+      content: "";
+      position: absolute;
+      top: 2px;
+      left: 2px;
+      height: 26px;
+      width: 26px;
```

Comment:
If you create variables for top and left you can calculate height and width here.